src/components/shared/AmountItem/AmountItem.js

```diff
@@ -34,14 +34,17 @@ class AmountItem extends Component {
     const changeAmount = `${change} %`;
     return (
       <Trend>
-        <SecondaryAmount>{changeAmount}</SecondaryAmount>
-        <IconChangeType>
-          <Icon
-            color={changeIconName.color}
-            name={changeIconName.name}
-            size={14}
-          />
-        </IconChangeType>
+        <SecondaryAmount>{(change && changeAmount) || '-- %'}</SecondaryAmount>
```

Comment:
Let's simplify this a bit. 

We can move this formatting to `changeAmount` declaration above and turn it into something like 

```jsx
const changeAmount = change ? `${change} %` : '-- %';

...

<SecondaryAmount>{changeAmount}</SecondaryAmount>
```

or 

```jsx
const changeAmount = change || '--';

...

<SecondaryAmount>{changeAmount} %</SecondaryAmount>
```

Btw, what happens here if `amount` value is `0.00`?